docs/supported-clients.md

```diff
@@ -8,17 +8,18 @@ This is the list clients that we have been using successfully while developing a
 
 | Client                | Type    | Oauth | Text | Images | Search | Notifs | Edit<br>profile | Source       | Works?
 | :-------------------- | :------ | :---- | :--- | :----- | :----- | :----- | :-----------    | :----------- | :---
-| [Pinafore][1]         | Web     | yes   | yes  | yes    | yes    | yes    | n/a             | [github][2]  | ✅
+| [Pinafore][1]         | Web     | yes   | yes  | yes    | yes    | yes    | n/a             | [github][2]  | ✅ <sup>1</sup>
 | [Mastodon Android][4] | Mobile  | yes   | yes  | yes    | yes    | yes    | yes             | [github][5]  | ✅
 | [Mastodon iOS][3]     | Mobile  | yes   | yes  | yes    | yes    | yes    | yes             | [github][6]  | ✅
-| [Ivory for iOS][7]    | Mobile  | yes   | ?    | ?      | ?      | ?      | ?               | n/a          | ❌ <sup>1</sup>
-| [Elk][8]              | Web     | yes   | yes  | yes    | yes    | yes    | yes             | [github][9]  | ✅ <sup>2</sup>
-| [Tooot][10]           | Mobile  | yes   | yes  | yes    | yes    | yes    | n/a             | [github][11] | ✅ <sup>2</sup>
-| [Mammoth][10]         | Mobile  | yes   | yes  | yes    | yes    | yes    | n/a             | [github][11] | ✅ <sup>2</sup> <sup>3</sup>
+| [Ivory for iOS][7]    | Mobile  | yes   | ?    | ?      | ?      | ?      | ?               | n/a          | ❌ <sup>2</sup>
+| [Elk][8]              | Web     | yes   | yes  | yes    | yes    | yes    | yes             | [github][9]  | ✅ <sup>3</sup>
+| [Tooot][10]           | Mobile  | yes   | yes  | yes    | yes    | yes    | n/a             | [github][11] | ✅ <sup>3</sup>
+| [Mammoth][12]         | Mobile  | yes   | yes  | yes    | yes    | yes    | n/a             | n/a          | ✅ <sup>3</sup> <sup>4</sup>
 
-<sup>1</sup> we know what the issue is and are in contact with the author<br/>
-<sup>2</sup> [conversations][conversations], [favourites][favorites], [bookmarks][bookmarks] and [trends][trends] are not implemented yet<br/>
-<sup>3</sup> notification filters are broken
+<sup>1</sup> [unmaintained][pinafore-unmaintained]<br/>
```

Comment:
this sounds like Wildebeest isn't maintaining compatiblity with pinafore. Can we rephrase that to `client unmaintained`?

Reply:
definitely! I'll get right on it